pkg/cryptoinfo/info.go

```diff
@@ -13,18 +13,52 @@ type KeyInfo struct {
 
 // Maybe make these types?
 const (
-	kiPEM         = "PEM"
-	kiDER         = "DER"
-	kiCertificate = "CERTIFICATE"
+	kiPEM = "PEM"
+	kiDER = "DER"
+	kiP12 = "P12"
+
+	kiCACERTIFICATE = "CA-CERTIFICATE" // What is correct here?
+	kiCaCertificate = "certificate"
+	kiCERTIFICATE   = "CERTIFICATE"
+	kiCertificate   = "certificate"
+	kiKEY           = "KEY"
+	kiKey           = "key"
```

Comment:
hm. these variables are.. interesting. maybe fine.

Interesting, because the names of the vars (even when coupled with the values) aren't super informative. Maybe making them types would improve the situation? Not sure.

Reply:
They feel like an uneasy compromise to me. They're not really needed -- I could just put them into the various `New` routines, but it felt nice to use constants instead.

I could convert them to types, it would provide the desired type safety... Hrm